.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+node_js:
+  - stable
+
+
+cache:
+  directories:
+    - "node_modules"
+
+before_install:
+  - npm install
+script:
+  - npm run coverage
```

Comment:
I remember you said that you are now using yarn but npm is will be used on Travis